### PR TITLE
[EPMTIOOPS-8059] [EPAM HEROES | Cirro-ruby-client] Create client for creation hero event

### PR DESCRIPTION
**Pull request by vv-developer**
…creation hero event

### CODE REVIEW

File: spec/cirro_io_v2/resources/epam_heroes_spec.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        grantee_id: '1',
+        comment: 'MyComment',
+        event: 'MyEvent',
+        grantor_id: '2',
+        cc_emails: ['Name_Surname@example.com', 'test@test.com']
+      }
+    end
+
+    it 'creates a epam_heroes' do
+      stub_api = stub_request(:post, "#{site}/v2/epam_heroes")
```
Comment (rostikkkk2) [inline spec/cirro_io_v2/resources/epam_heroes_spec.rb:22]: move it pls to let

File: spec/cirro_io_v2/resources/epam_heroes_spec.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
+RSpec.describe CirroIOV2::Resources::EPAMHeroes do
+  let(:site) { 'http://api.cirro.io' }
+  let(:employee_id) { '1' }
```
Comment (rostikkkk2) [inline spec/cirro_io_v2/resources/epam_heroes_spec.rb:3]: remove this line

File: spec/cirro_io_v2/resources/epam_heroes_spec.rb [hunk 3]
```diff
@@ -0,0 +1,39 @@
+RSpec.describe CirroIOV2::Resources::EPAMHeroes do
```
Comment (rostikkkk2) [inline spec/cirro_io_v2/resources/epam_heroes_spec.rb:1]: EpamHeroes

File: spec/cirro_io_v2/resources/epam_heroes_spec.rb [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+      expect(epam_heroes.class).to eq(CirroIOV2::Responses::EPAMHeroesResponse)
+      expect(epam_heroes.object).to eq('epam_heroes')
+      expect(epam_heroes.token).to be_present
+      expect(epam_heroes.comment).to eq(params[:comment])
+      expect(epam_heroes.comment).to be_present
+      expect(epam_heroes.event).to eq(params[:event])
+      expect(epam_heroes.event).to be_present
+      expect(epam_heroes.assigner_type).to eq(params[:assigner_type])
+      expect(epam_heroes.cc_emails).to eq(params[:cc_emails])
+    end
```
Comment (rostikkkk2) [inline spec/cirro_io_v2/resources/epam_heroes_spec.rb:37]: you need to check response entities from your struct response, that there are the same as in your fixtures

File: spec/cirro_io_v2/resources/epam_heroes_spec.rb (hunk 1 above)
Comment (vv-developer) [reply spec/cirro_io_v2/resources/epam_heroes_spec.rb]: done

File: spec/cirro_io_v2/resources/epam_heroes_spec.rb (hunk 3 above)
Comment (vv-developer) [reply spec/cirro_io_v2/resources/epam_heroes_spec.rb]: done

File: spec/cirro_io_v2/resources/epam_heroes_spec.rb (hunk 4 above)
Comment (vv-developer) [reply spec/cirro_io_v2/resources/epam_heroes_spec.rb:35]: Thank you for the review. 
All done!

**Review by rostikkkk2 — APPROVED** [review]

**Review by garciaf — APPROVED** [review]

**Pull request merged**